### ISSUE TITLE
registration email goes to spam

### ISSUE THREAD

**Issue by nconrad** [PATRIC3/patric3_website#1678]
I think this may have been reported before, but I couldn't find a ticket.  

After I registered a new user with a gmail email address, the registration email went  spam.  Google is suggesting this: https://support.google.com/mail/answer/81126?hl=en#authentication

**Comment by olsonanl** [comment]
This may resolve when we move user service to ANL and outgoing mail is sent through the authenticated ANL mail servers

**Comment by rkenyon** [comment]
Assigned Dan Olson.  Not sure if that is the right person, but will probably need to coordinate through him once user service is moved.

**Comment by danielolson5** [comment]
Yes, I just discovered this week that the only email servers we have listed as trusted belong to a mass mailing service.  

@nconrad @dmachi @olsonanl Do any of you know what IP address the new account emails currently originate from?

**Comment by danielolson5** [comment]
I created a test account and it is in the message header - 128.173.97.24 (almaren.bioinformatics.vt.edu).

**Comment by olsonanl** [comment]
This is the other issue that was triggering my worries about using ANL mail servers. $15/month from socketlabs.com ...

**Comment by danielolson5** [comment]
I updated the SPF record for patricbrc.org so VBI and ANL networks are trusted.  I'll test this again in a few hours to see if google trusts the registration notices now. 

To solve this with socketlabs.com I believe we would need to use them as our mail gateway - that might be more trouble than it is worth.
  
I'm ok with maintaining our DNS records, now that I know this is an issue.

**Comment by olsonanl** [comment]
Yes, we’d configure whatever code sends mail to use that as the authenticated gateway. I have had zero bounces since I started using that (sending hundreds of messages to a lot of disparate addresses every month)

—bob

On Sep 14, 2017, at 3:24 PM, Daniel Olson <notifications@github.com<mailto:notifications@github.com>> wrote:


I updated the SPF record for patricbrc.org<http://patricbrc.org> so VBI and ANL networks are trusted. I'll test this again in a few hours to see if google trusts the registration notices now.

To solve this with socketlabs.com<http://socketlabs.com> I believe we would need to use them as our mail gateway - that might be more trouble than it is worth.

I'm ok with maintaining our DNS records, now that I know this is an issue.
